### DISCUSSION TITLE
doker部署出错

### DISCUSSION THREAD

**Discussion by api81244390** [messense/aliyundrive-webdav#392] [Q&A] [answered]
### 问题描述

启动几秒后就失败
doker提示Restrating   ports为空

### 重现步骤

端口未占用
token确定没问题

### 版本

Debian10

### 运行平台

docker

### 日志

{"log":"\u001b[2m2022-04-19T08:37:27.270133Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:27.270266702Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:27.270691188Z"}
{"log":"\u001b[2m2022-04-19T08:37:28.673569Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:28.673722724Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:28.67404062Z"}
{"log":"\u001b[2m2022-04-19T08:37:29.955898Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:29.956073532Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:29.956563171Z"}
{"log":"\u001b[2m2022-04-19T08:37:31.438918Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:31.439172765Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:31.439395001Z"}
{"log":"\u001b[2m2022-04-19T08:37:33.339905Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:33.340275883Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:33.340363948Z"}
{"log":"\u001b[2m2022-04-19T08:37:37.125043Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:37.12520148Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:37.125518755Z"}
{"log":"\u001b[2m2022-04-19T08:37:41.315721Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:41.315962491Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:41.316236205Z"}
{"log":"\u001b[2m2022-04-19T08:37:48.667733Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:37:48.667875933Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:37:48.668287615Z"}
{"log":"\u001b[2m2022-04-19T08:38:02.650363Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:38:02.650663307Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:38:02.651493174Z"}
{"log":"\u001b[2m2022-04-19T08:38:29.554294Z\u001b[0m \u001b[31mERROR\u001b[0m \u001b[2maliyundrive_webdav::drive\u001b[0m\u001b[2m:\u001b[0m refresh token failed: HTTP status client error (400 Bad Request) for url (https://websv.aliyundrive.com/token/refresh): {\"code\":\"InvalidParameter.ClientId\",\"message\":\"The input parameter client_id is not valid. \",\"requestId\":null}\n","stream":"stdout","time":"2022-04-19T08:38:29.554441486Z"}
{"log":"Error: get default drive id failed\n","stream":"stderr","time":"2022-04-19T08:38:29.555014741Z"}

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/discussions/210
